snap/snapcraft.yaml

```diff
@@ -2,9 +2,24 @@ name: srsran
 base: core20
 version: '20.10'
 summary: srsRAN is a 4G/5G software radio suite developed by SRS.
-description: srsRAN is a 4G/5G software radio suite developed by SRS.
+description: |
+  srsRAN is a free and open-source 4G and 5G software radio suite.
+
+  Featuring both UE and eNodeB/gNodeB applications, srsRAN can be used with third-party core network solutions to build complete end-to-end mobile wireless networks. For more information, see https://www.srsran.com/.
+
+  The srsRAN suite currently includes:
+  - srsUE: a full-stack 4G and 5G NSA / SA UE application
+  - srsENB: a full-stack 4G eNodeB with 5G NSA / SA gNodeB capabilities
+  - srsEPC: a light-weight 4G EPC implementation with MME, HSS and S/P-GW
+
+  All srsRAN software runs in linux with off-the-shelf compute and radio hardware.
 grade: devel
-confinement: devmode
+confinement: classic
```

Comment:
based on the [documentation](https://snapcraft.io/docs/snap-confinement), it's not clear to me that devmode and classic provide the same level of access.
I'm testing it and I see that the behaviour is not the same.
Please don't merge yet.

Reply:
Reverted to using devmode